cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -432,42 +435,81 @@ var (
 		},
 	}
 	gatewaysConnectionStats = &cobra.Command{
-		Use:     "get-connection-stats [gateway-id]",
+		Use:     "get-connection-stats [gateway-ids]",
 		Aliases: []string{"connection-stats", "cnx-stats", "stats"},
-		Short:   "Get connection stats for a gateway",
+		Short:   "Get connection stats for a (group of) gateway(s).",
+		Long: `Get connection stats for a (group of) gateway(s).
+The command arguments support multiple gateway IDs.
+Alternatively use the --gateway-ids flag.
+If both the parameter and the flag are provided, the flag is ignored.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			gtwID, err := getGatewayID(cmd.Flags(), args, true)
+			var (
+				ids = args
+				err error
+			)
+			if len(ids) == 0 {
+				raw, err := cmd.Flags().GetString("gateway-ids")
```

Comment:
Couldn't this be `GetStringSlice`?

Reply:
Please see https://github.com/TheThingsNetwork/lorawan-stack/pull/5536#discussion_r920080643

Reply:
Getting the flag as a string slice would actually make sense

Reply:
I'm falling to see how the comment relates to the usage of `GetStringSlice` on the flag.  It should work with just one value and would be fetched only when ids aren't passed as an argument.

Reply:
`gateway-ids` is an alias to `gateway-id` which is an existing flag.

```
./ttn-lw-cli gtw get-connection-stats --help
Get connection stats for a (group of) gateway(s).
The command arguments support multiple gateway IDs.
Alternatively use the --gateway-ids flag.
If both the parameter and the flag are provided, the flag is ignored.

Usage:
  ttn-lw-cli gateways get-connection-stats [gateway-ids] [flags]

Aliases:
  get-connection-stats, connection-stats, cnx-stats, stats

Flags:
      --gateway-eui string
      --gateway-id string
      --gateway-ids string
```

If I reverse that alias to use `StringSlice`

```
Flags:
      --gateway-id strings    comma separated list of gateway IDs to batch get stats (default [])
      --gateway-ids strings   comma separated list of gateway IDs to batch get stats
```

You're either wrong labelling `gateway-id` as a slice or `gateway-ids` as a string value. I don't really prefer one over the other. If I really have to use `StringSlice` here, I can. But I don't think it adds much.